src/covid/pages/lab-results.encounter-list.tsx

```diff
@@ -178,21 +178,21 @@ const columnsPending: EncounterListColumn[] = [
         form: { name: 'covid_lab_test', package: 'covid' },
         encounterUuid: encounter.uuid,
         intent: '*',
-        label: 'View Lab Test',
+        label: 'View Details',
         mode: 'view',
       },
       {
-        form: { name: 'covid_lab_result', package: 'covid' },
+        form: { name: 'covid_sample_collection', package: 'covid' },
         encounterUuid: encounter.uuid,
         intent: '*',
-        label: 'Edit Lab Result',
+        label: 'Collect Sample',
         mode: 'edit',
       },
       {
-        form: { name: 'covid_lab_order_cancellation', package: 'covid' },
+        form: { name: 'covid_lab_result', package: 'covid' },
```

Comment:
don't these name changes need a corresponding change of the files in the packages